website/docs/d/database_connection.html.markdown

```diff
@@ -0,0 +1,324 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_database_connection"
+description: |-
+  Get information about database_connection
+subcategory: "The IBM Cloud Databases API"
+---
+
+# ibm_database_connection
+
+Provides a read-only data source for database_connection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_database_connection" "database_connection" {
+	endpoint_type = "public"
+	id = "id"
```

Comment:
provide some valid interpolation syntax like id can be read from a databse resource
id = ibm_database.id
or 
id = data.ibm_database.id

Reply:
Fixed